Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * Commandline Interface: Enable output of storage layout with `--storage-layout`.
 
 Bugfixes:
+ * General: Restrict the size of dynamic memory arrays to 64 bits during creation fixing a possible overflow.
```

Comment:
For buglist entries, we usually use a category "Important Bugfixes".

Would add "at runtime".